kvrocks.conf

```diff
@@ -335,6 +335,20 @@ json-max-nesting-depth 1024
 # Default: json
 json-storage-format json
 
+# Whether to enable transactional mode engine::Context.
+#
+# If enabled, is_txn_mode in engine::Context will be set properly,
+# which is expected to improve the consistency of commands.
+# If disabled, is_txn_mode in engine::Context will be set to false,
+# making engine::Context equivalent to engine::Storage.
+#
+# NOTE: This is an experimental feature. If you find errors, performance degradation,
+# excessive memory usage, excessive disk I/O, etc. after enabling it, please try disabling it.
+# At the same time, we welcome feedback on related issues to help iterative improvements.
+# 
+# Default: no
+txn-context-enabled no
```

Comment:
maybe we can add some go cases for "yes" mode.

Reply:
Yeah, we need to test, but "some" may actually be almost all.
Is it possible to add a "yes" mode process to the GitHub Actions?

Reply:
You can just change the go cases to make some test functions like `TestXX` work on both mode, e.g.
```
TestXXAllMode() {
  TestXX(yes)
  TestXX(no)
}
```

Reply:
Okay, how about adding these tests in another PR? I need to go out recently, so I may not be able to update immediately. You can merge this PR first, and then I will create an issue to track it and solve it when I have free time.